doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -190,15 +192,40 @@ In a worst case scenario, the store time can be up to several minutes.
 This can for example happen when storing a large size replay protection list.
 It is recommended to configure the settings subsystem's internal caches to improve the performance.
 
-* The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+
+* :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`
+
+The size of the Settings NVS name cache, :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE_SIZE`, is recommended to be at least equal to the number of setting entries the device is expected to store. These entries may include entries in the RPL, whose size is given by :kconfig:option:`CONFIG_BT_MESH_CRPL`, :ref:`configuration values <ug_bt_mesh_configuring_optional_features>`, application- and network keys, application specific states etc. Using :ref:`bluetooth_mesh_sensor_server` as an example configured according to the sample's :ref:`configuration guide <bluetooth_mesh_sensor_server_conf_models>`, this has the following possible entries:
```

Comment:
I'd suggest to have a generic section. Most of the list items starting from line 201 could be moved into that section. It could say _something like_:
```
The mesh stack stores the following data persistently (each bullet-point is a separate setting entry):
- Network information (primary address and device key)
- Configuration parameters (features support, default TTL, Network Transmit and Relay Retransmit states)
- IV index
- Sequence number
- Heartbeat publication state
- Application key (see :kconfig:option:`CONFIG_BT_MESH_APP_KEY_COUNT`)
- Network key (see :kconfig:option:`CONFIG_BT_MESH_SUBNET_COUNT`)
- Label UUIDs for virtual addressing (see :kconfig:option:`BT_MESH_LABEL_COUNT`)
- RPL entry (see :kconfig:option:`CONFIG_BT_MESH_CRPL`)

The following data is stored by the mesh stack per each model (each bullet-point is a separate setting entry):
- Model subscription state
- Model publication state
- Bound application keys
- Subscription list (group and virtual addresses)
- Label UUIDs the model is subscribed to
- Model specific data
  - The information about a model specific data stored persistently can be found under the ``Persistent storage`` section of corresponding model documentation.
```

And then we can provide example:
```
Using :ref:`bluetooth_mesh_sensor_server` as an example, this has the following possible entries:
- 32 RPL entries
- 3 application keys
...
etc.
```
\+ mention the model specific data and the sample specific data (btw, we have `Persistent storage` section in each model saying what each models store, e.g.: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/bluetooth_services/mesh/sensor_srv.html#persistent-storage, so we can refer to in in the example).

Reply:
Added generic section. In the example I only mention the generics which possibly has variable amount of entries